packages/nodes-base/nodes/Mattermost/v1/actions/user/getAll/execute.ts

```diff
@@ -95,4 +95,4 @@ export async function getAll(this: IExecuteFunctions, index: number): Promise<IN
 	}
 
 	return this.helpers.returnJsonArray(responseData);
-}
\ No newline at end of file
+}
```

Comment:
In this file we missed the important info - there are 3 instances of error thrown that could have this info

Reply:
Addressed, thank you.